scala_zio_bench/src/main/protobuf/helloworld.proto

```diff
@@ -0,0 +1,39 @@
+// Copyright 2015 gRPC authors.
```

Comment:
let's copy this guy from proto/ in the dockerfile, like in e.g. Java implementations

Reply:
In fact it seems you're already doing that, not sure why this file is added?

Reply:
I added the file so I can develop locally (i.e build outside the docker container), but forgot to remove it...